CHANGELOG.md

```diff
@@ -1,9 +1,20 @@
+## v2.1.2
+### 2023-11-14
+
+This version of `earthdata-varinfo` updates the value of the `LongName`
+attribute in generated UMM-Var records to use the value of the CF-Convention
+`long_name` attribute for a variable, if it is present in the file. If this
+attribute is not present in the in-file metadata, then the full path to the
```

Comment:
Might be a super nit:
Is it the full path name that was used before? I thought the full path usually started with a `/`

Reply:
The default is still the value that was used before, which is the full path (stripped of the leading `/` to conform with UMM-Var expectations). I'll clarify this comment and then merge. Good catch.